predicators/utils.py

```diff
@@ -166,6 +166,24 @@ def segment_trajectory_to_atoms_sequence(
     return atoms_seq
 
 
+def order_data(
+    trajectories: List[LowLevelTrajectory],
+    ground_atom_dataset: List[GroundAtomTrajectory], rng: np.random.Generator
+) -> Tuple[List[LowLevelTrajectory], List[GroundAtomTrajectory]]:
+    """Re-orders a list of trajectories and matching list of ground atoms,
+    using the np.random.Generator provided.
+
+    This function returns another list of trajectories and matching list
+    of ground atoms in a new order.
+    """
+    rnd_seed = int(rng.random() * 1000000)
```

Comment:
this way looks a little weird to me. can we instead use the pattern here? https://stackoverflow.com/questions/23289547/shuffle-two-list-at-once-with-same-order

Reply:
actually, now that I see where this is being used, maybe we don't really need a utility function for this and we can just move this code inside the loop? the fact that it's hard to come up with a specific name is good evidence that we probably won't be using this utility elsewhere. and since it's short, might as well just put it where it's used.